### PR TITLE
add back face culling option

### PR DESCRIPTION
**Pull request by annaborn**
@planthaber Steffen, could you please take a look and merge it, if everything is ok? Thank you! :)

an option backculling is added into viewer options

### CODE REVIEW

**Comment by planthaber** [comment]
hmm, wouldn't it make more sense to add this option to the plugin itself?
I know it is about the MLS Visualization from https://github.com/envire/slam-maps, so we could also add it as a property of the MLSVisualization: https://github.com/envire/slam-maps/blob/master/viz/MLSMapVisualization.hpp#L68

**Comment by planthaber** [comment]
Then, please also add it as a property here: https://github.com/rock-core/gui-vizkit3d/blob/master/src/Vizkit3DWidget.hpp#L160, so we can en- and disable it from the propertybrowser

File: src/Vizkit3DWidget.hpp [hunk 1]
```diff
@@ -158,6 +159,7 @@ namespace vizkit3d
             Q_ENUMS( CAMERA_MANIPULATORS )
             Q_PROPERTY( QStringList manipulator READ getAvailableCameraManipulators WRITE setCameraManipulator )
             Q_PROPERTY( double transformerTextSize READ getTransformerTextSize WRITE setTransformerTextSize );
+            Q_PROPERTY( bool backCulling READ isBackCulling WRITE setBackCulling);
```
Comment (annaborn) [inline src/Vizkit3DWidget.hpp:162]: @planthaber  the backCulling property in Vizkit3DConfig is already here :)

**Comment by annaborn** [comment]
> Then, please also add it as a property here: https://github.com/rock-core/gui-vizkit3d/blob/master/src/Vizkit3DWidget.hpp#L160, so we can en- and disable it from the propertybrowser

@planthaber  It is already there :) https://github.com/rock-core/gui-vizkit3d/pull/71/files#r1155616171

File: src/Vizkit3DWidget.hpp (hunk 1 above)
Comment (planthaber) [reply src/Vizkit3DWidget.hpp:162]: ok, overlooked that one

**Pull request merged**